halo2_proofs/src/dev.rs

```diff
@@ -355,6 +357,25 @@ impl<F: Group + Field> Mul<F> for Value<F> {
     }
 }
 
+impl<F: Group + Field> ConstantTimeEq for Value<F> {
+    #[inline]
+    fn ct_eq(&self, rhs: &Self) -> Choice {
```

Comment:
This isn't actually constant-time, so couldn't be accepted upstream. (I'm aware you're just testing at the moment.)